docs/release-notes/8.4.asciidoc

```diff
@@ -0,0 +1,71 @@
+* Enhances rule previews with configurable rule intervals and look-back times ({pull}137102[#137102]).
+* Enhances the `status pending` badge for endpoint actions with a detailed status when you hover on it ({pull}136966[#136966]).
+* Turns grouped navigation on by default ({pull}136819[#136819]).
+* Adds a confirmation dialog to bulk rule export ({pull}136418[#136418]).
+* Adds index pattern information to the Inspect panel ({pull}136407[#136407]).
+* Adds a custom dashboards table to the Dashboards page ({pull}136221[#136221], {pull}136671[#136671]).).
+* Fixes a performance issue with alerts that have large fields ({pull}135956[#135956]).
+* Updates the rule exceptions UI ({pull}135255[#135255]).
+* Fixes performance issues with rules management ({pull}135311[#135311]).
+* Allows you to define a fallback `@timestamp` when you've defined a timestamp override ({pull}135116[#135116]).
+* Enhances the host risk score modal UI ({pull}133708[#133708]).
+* Updates the lists index template to use new logic ({pull}133067[#133067]).
+* Adds event filters to event correlation rules ({pull}132507[#132507]).
+* Allows you to define a data view or index pattern for rules ({pull}130929[#130929]).
+* Adds a treemap and multi-dimensional alert grouping to the Alerts page ({pull}126896[#126896]).
+* Fixes an incorrect counter for exported rules ({pull}138598[#138598]).
+* Fixes event filters based on OS version ({pull}138517[#138517]).
+* Fixes a bug that could change the batch size for event search in indicator rules ({pull}138356[#138356]).
+* Fixes a bug that could crash the Alert details flyout ({pull}138331[#138331]).
+* Fixes the preview button for {ml} rules ({pull}137878[#137878]).
+* Fixes a bug that could crash the Endpoints list when a policy ID was missing ({pull}137788[#137788]).
+* Fixes a bug that could interfere with opening host or user details pages ({pull}137719[#137719]).
+* Fixes several bugs related to refreshing the Alerts page ({pull}137620[#137620]).
+* Fixes a bug with bulk rule deletion with a defined data view ({pull}137585[#137585]).
+* Fixes a bug with look-back time in the **Advanced query preview** ({pull}137517[#137517]).
+* Fixes a bug that prevented Threshold Rules' Timeline templates from being respected during investigations ({pull}137233[#137233]).
+* Fixes a permissions bug related to the **Save Timeline** button ({pull}136724[#136724]).
+* Fixes a bug with selecting Timeline templates with the same name ({pull}135694[#135694]).
+* Fixes field aliases to `signal-threshold_result.*` ({pull}135565[#135565]).
+* Fixes a bug that lost track of which rules you had selected after refreshing the Rules page ({pull}135533[#135533]).
+* Fixes a bug that lost track of which rules you had selected after applying a bulk action on the Rules page ({pull}135291[#135291]).
+* Fixes a bug that prevented the rule details table from pausing auto-refresh while bulk actions are being applied ({pull}135208[135208]).
```

Comment:
I think this is just the Rules table, not details:
```suggestion
* Fixes a bug that prevented the Rules table from pausing auto-refresh while bulk actions are being applied ({pull}135208[135208]).
```

Reply:
Yep, you're correct @joepeeples .